wrapper/src/main/java/software/amazon/jdbc/PropertyDefinition.java

```diff
@@ -179,6 +186,10 @@ public class PropertyDefinition {
     return PROPS_BY_NAME.get(name);
   }
 
+  public static @Nullable AwsWrapperProperty byNameIgnoreCase(final String name) {
+    return PROPS_BY_NAME_LOWERCASE.get(name.toLowerCase());
+  }
+
```

Comment:
Instead of a new function can we overload the original `byName`?
```java
  public static @Nullable AwsWrapperProperty byName(final String name) {
    return byName(name, false);
  }

  public static @Nullable AwsWrapperProperty byName(final String name, final boolean ignoreCase) {
    return ignoreCase ? PROPS_BY_NAME_LOWERCASE.get(name.toLowerCase()) : PROPS_BY_NAME.get(name);
  }
```
If we do this we can simplify the logic in ConnectionUrlParser